src/bindings/python/CMakeLists.txt

```diff
@@ -130,6 +130,32 @@ endfunction()
 
 ov_check_python_build_conditions()
 
+# check __init__.py files alignment
+
+function(ov_check_init_files_alignment)
+    # check the files in pairs
+    list(APPEND init_files
+        "${OpenVINOPython_SOURCE_DIR}/src/openvino/__init__.py"
+        "${OpenVINOPython_SOURCE_DIR}/src/compatibility/openvino/__init__.py")
```

Comment:
Add `openvino-dev` inits as well. Just to have a full scope of validation between both.

Reply:
I suppose we need to add only one from `openvino-dev`, because similarity of those files are checked in `tools/openvino_dev/CMakeLists.txt`

Reply:
I added full validation to both CMake files. The time it takes to check the files is negligible and that way we have an explicit, clear check.